pymeasure/experiment/procedure.py

```diff
@@ -284,6 +284,9 @@ def emit(self, topic, record):
     def should_stop(self):
         raise NotImplementedError('should be monkey patched by a worker')
 
+    def has_next(self):
```

Comment:
I realise this is also not done for the other monkeypatched methods, but it would be good to add a docstring to specify what the method does (or rather, what it returns) and thereby ensure (I presume at least) that the functionality/method is included in the documentation.

Reply:
I added a docstring to the new function.